pdr_backend/pdr_dashboard/assets/styles.css

```diff
@@ -30,6 +30,8 @@ body {
   margin: 0px;
   padding: 0px;
   width: 100%;
+  padding-left: 20px;
+  padding-right: 20px;
```

Comment:
we want to have this on the entire page content, so we can move this to the html body